pumpkin/src/world/mod.rs

```diff
@@ -192,7 +192,6 @@ impl World {
         {
             let mut abilities = player.abilities.lock().await;
             abilities.allow_flying = true;
-            abilities.flying = true;
         }
         player.send_abilties_update().await;
 
```

Comment:
At this point you we can actually just not send the abilities at all

Reply:
if we dont want survival mode player to allow flight we can remove the mutating of abilities here. But there are still other variables that need to be sent